src/lib.rs

```diff
@@ -95,16 +95,16 @@ impl Color for RGB {
     }
 
     fn to_rgba(self) -> RGBA {
-        RGBA::new(self.r, self.g, self.b, 1.0)
+        RGBA::new(self.r, self.g, self.b, 255)
     }
 }
 
 #[derive(Debug, Copy, Clone, PartialEq)]
 /// A struct to represent how much red, green, and blue should be added to create a color.
 /// Also handles alpha specifications.
 ///
-/// Valid values for r, g, and b must fall between `0-255`.
-/// Valid values for alpha specifications must be a percentage ranging between `0.0 - 1.0`.
+/// Valid values for r, g, b, and a must fall between `0-255`.
+/// Converts alpha specifications from a u8 integer into a percentage ranging between `0.00 - 1.00`.
```

Comment:
I think we can probably just ✂️ this line now. The "conversion" only happens in `to_css` now